src/FluentHub.Octokit/GlobalUsings.cs

```diff
@@ -2,22 +2,20 @@
 // Licensed under the MIT License. See the LICENSE.
 
 // Default usings
-global using global::System;
-global using global::System.Collections.Generic;
-global using global::System.Linq;
-global using global::System.Linq.Expressions;
-global using global::System.Runtime.Serialization;
-global using global::System.Text;
-global using global::System.Threading.Tasks;
-global using global::Newtonsoft.Json;
-global using global::Newtonsoft.Json.Converters;
-
 // Custom usings
```

Comment:
```suggestion
```

As you can see, doing this has messed up some notes surrounding import statements (see the comment on https://github.com/0x5bfa/FluentHub/pull/510/commits/d8b4c221751925ab7a432a19989149b9cdd5669a)

I'll just remove this comment and leave it in the pretty formatted way.

Reply:
Fixed in [`beb88cd`](https://github.com/0x5bfa/FluentHub/pull/510/commits/beb88cdeb2f270f4330a014b23d1b468150a1e1a)